src/generators/web.cr

```diff
@@ -10,8 +10,7 @@ class LuckyCli::Generators::Web
     project_name : String,
     @options : Options
   )
-    @project_dir = project_name.gsub(" ", "_")
-    @project_name = @project_dir.gsub("-", "_")
```

Comment:
The reason these were different is that it would keep the project directory as you typed it (for example `crystal-mastery`), but would use an underscore in the project_name, because that it what is used in the shard.yml and it will fail with a dash. I think that would still be nice to keep distinct so that you can have control over the project directory name.